src/components/BeamContext.tsx

```diff
@@ -81,17 +82,19 @@ export function BeamProvider({ children, ...presentationProps }: BeamProviderPro
   return (
     <BeamContext.Provider value={{ ...context }}>
       <PresentationProvider {...presentationProps}>
-        <AutoSaveStatusProvider>
-          <SnackbarProvider>
-            {/* OverlayProvider is required for Modals generated via React-Aria */}
-            <OverlayProvider>
-              {children}
-              {/*If the drawer is open, assume it will show modal content internally. */}
-              {modalRef.current && drawerContentStackRef.current.length === 0 && <Modal {...modalRef.current} />}
-            </OverlayProvider>
-            <SuperDrawer />
-          </SnackbarProvider>
-        </AutoSaveStatusProvider>
+        <HbSpinnerProvider>
```

Comment:
**thought:** I'm wondering if it really makes sense to include this in the `BeamContext`. Thinking of it from "Beam is only used by Homebound", then yes, I could see how it makes sense. But since Beam is open sourced, and maybe one day folks outside of Homebound will use Beam (🤷), then I'd lean against automatically including this spinner context to the rest of the Beam contexts.

Reply:
I can pull it out.

I'd wanted it to be usable out-of-the-box, where you _could_ use it inside your app to customize it, otherwise you get it for free. But the base `createContext()` even without a provider has everything it needs. I'm also not sure if/where I should drop a README to make it easier to set up.